code/datums/mutations/actions.dm

```diff
@@ -234,3 +235,94 @@
 
 	var/obj/item/bodypart/BP = pick(parts)
 	BP.dismember()
+
+/datum/mutation/human/overload
+	name = "Overload"
+	desc = "Allows an Ethereal to overload their skin to cause a bright flash."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>Your skin feels more crackly.</span>"
+	instability = 30
+	power = /obj/effect/proc_holder/spell/self/overload
+	species_allowed = list("ethereal")
+
+/obj/effect/proc_holder/spell/self/overload
+	name = "Overload"
+	desc = "Concentrate to make your skin energize."
+	clothes_req = FALSE
+	human_req = FALSE
+	charge_max = 400
+	action_icon_state = "blind"
+	var/maxdist = 4
+
+/obj/effect/proc_holder/spell/self/overload/cast(mob/user = usr)
+	if(!is_species(user, /datum/species/ethereal))
+		return
+
+	var/list/mob/targets = viewers(maxdist, get_turf(user))
+	targets -= user
+	visible_message("<span class='disarm'>[user] emits a blinding light!</span>")
+	for(var/mob/living/carbon/C in targets)
+		C.flash_act()
+		C.Paralyze(10 + (5*maxdist))
```

Comment:
this should be if(C.flash_act([i think some sort of intensity value goes here]))
right now, it bypasses flash protection


other than code issues, this is incredibly powerful, being an AOE stun. You should make it work like a flashbang instead